src/components/tables/Captures/index.tsx

```diff
@@ -15,7 +15,7 @@ const queryColumns = [
     'catalog_name',
     'connector_image_name',
     'connector_image_tag',
-    'connector_open_graph->en-US->>image',
```

Comment:
This is not the image path in terms of what docker image to pull... this is the image that we display as the icon.